core/src/main/java/org/pentaho/di/metastore/MetaStoreConst.java

```diff
@@ -72,7 +75,7 @@ public MetastoreLocator get() {
               metastoreLocator = metastoreLocators.stream().findFirst().orElse( null );
             }
           } catch ( KettlePluginException e ) {
-            LogChannel.GENERAL.logError( "Error getting metastore locator", e );
+            logger.error( "Error getting metastore locator", e );
```

Comment:
Why we can't continue to use `LogChannel.GENERAL.logError` ?

Reply:
The pentaho-platform tests were failing because of something like "General LogChannel not initialized". This error caused the tests to crash. I think either we never hit this exception block in the pentaho-kettle build, *or* the LogChannel stuff is somehow initialized in those cases. 
In either case, this type of logging seems more common. I also tried some other things to get the LogChannel stuff to work, but couldn't get it working. MetaStoreConst being a static-only class added complication.